x-pack/plugins/index_management/common/lib/template_serialization.ts

```diff
@@ -74,7 +74,7 @@ export function deserializeTemplate(
     indexPatterns: indexPatterns.sort(),
     template,
     ilmPolicy: settings?.index?.lifecycle,
-    composedOf,
```

Comment:
ES now sometimes omits the property in the response, so we could default it to an empty array just in case